paymentsheet/src/main/java/com/stripe/android/paymentelement/AnalyticEventCallback.kt

```diff
@@ -0,0 +1,55 @@
+package com.stripe.android.paymentelement
+
+import dev.drewhamilton.poko.Poko
+
+/**
+ * Called when an analytic event is emitted.
+ *
+ * These events are intended to be used for analytics purposes ONLY.
+ *
+ * @param event the [AnalyticEvent] that was emitted
+ */
+@ExperimentalAnalyticEventCallbackApi
+fun interface AnalyticEventCallback {
+    fun onEvent(event: AnalyticEvent)
+}
+
+@ExperimentalAnalyticEventCallbackApi
+abstract class AnalyticEvent internal constructor() {
+
+    @Override
+    override fun toString(): String {
```

Comment:
Is this only used for `PresentedSheet`? I think we probably want to implement these with poko or something similar. We also want this type to have an equals/hashcode as others do since they're generated with poko.